Frontend/ui-customer/src/Components/InputView/View.purs

```diff
@@ -0,0 +1,250 @@
+    , padding $ config.padding 
+    , background Color.squidInkBlue
+    , margin $ config.margin
+    , cornerRadius $ config.cornerRadius 
+    , onClick push $ const $ TextFieldFocusChanged config.id true
+    , stroke $ strokeValue 
+    ]
+    [  textView $ 
+          [ text $ config.textValue
+          , color config.textColor
+          , width MATCH_PARENT
+          , height WRAP_CONTENT
+          , maxLines 1
+          , ellipsize true
+          , gravity config.gravity
+          ] <> config.fontStyle -- (FontStyle.body6 LanguageStyle)
+     ]
+
+
+inputTextField :: forall w. (Action -> Effect Unit ) -> InputView -> PrestoDOM (Effect Unit) w
+inputTextField push config = 
+  linearLayout 
+    [ height WRAP_CONTENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , margin config.margin 
+    ] 
+    [ linearLayout
+      [ height WRAP_CONTENT
+      , width MATCH_PARENT
+      , background Color.squidInkBlue
+      , cornerRadius $ config.cornerRadius 
+      ][  editText $ 
+          [ height $ config.height 
+          , weight 1.0
+          , padding $ config.padding 
+          , color Color.black600
+          , gravity LEFT
+          , background Color.squidInkBlue
+          , cornerRadius $ config.cornerRadius 
+          , text config.textValue
+          , selectAllOnFocus true
+          , singleLine true
+          , hint config.placeHolder
+          , hintColor $ Color.blueGrey
+          , pattern "[^\n]*,255"
+          , id $ getNewIDWithTag $ config.id
+          , onChange (\action -> do 
+              void $ debounceFunction getDelayForAutoComplete push AutoCompleteCallBack config.isFocussed
+              void $ push action
+            ) InputChanged
+          , onFocus push $ const $ TextFieldFocusChanged config.id true
+          , cursorColor Color.yellow900
+          ] <> FontStyle.body6 LanguageStyle
+        , crossButtonView push config
+        ]
+    , bottomStrokeView $ boolToVisibility config.isFocussed
+    ]
+
+crossButtonView :: forall w. (Action -> Effect Unit) -> InputView -> PrestoDOM (Effect Unit) w
+crossButtonView push config = 
+      linearLayout
+        [ height $ V 32 
+        , width $ V 32
+        , gravity CENTER 
+        , padding $ config.clearTextIcon.padding 
+        , onClick push $ const $ ClearTextField config.id
+        , visibility $ boolToVisibility config.canClearText
+        ][  imageView
+            [ height $ config.clearTextIcon.height
+            , width $ config.clearTextIcon.width 
+            , imageWithFallback $ fetchImage FF_ASSET config.clearTextIcon.imageName
+            ]
+        ]
+
+bottomStrokeView :: forall w. Visibility -> PrestoDOM (Effect Unit) w
+bottomStrokeView bottomStrokeVisibility =
+  linearLayout
+  [ height $ V 1
+  , width MATCH_PARENT
+  , orientation HORIZONTAL
+  , visibility $ bottomStrokeVisibility
+  , background Color.white900
+  ][] 
+
+emptyTextView :: forall w. PrestoDOM (Effect Unit) w
```

Comment:
Can we create a common file in Utils for the commonly used view functions as these are being used everywhere